pkg/broker/ingress/ingress_handler_test.go

```diff
@@ -0,0 +1,94 @@
+package ingress
+
+import (
+	"context"
+	nethttp "net/http"
+	"net/url"
+	"testing"
+
+	cloudevents "github.com/cloudevents/sdk-go"
+	"go.uber.org/zap"
+
+	"github.com/cloudevents/sdk-go/pkg/cloudevents/transport/http"
+)
+
+var (
+	logger     = zap.NewNop()
+	channelURI = &url.URL{
+		Scheme: "http",
+		Host:   "testChannel",
+		Path:   "/",
+	}
+	brokerName = "testBroker"
+	validURI   = "/"
+	event      = cloudevents.NewEvent()
+	resp       = new(cloudevents.EventResponse)
```

Comment:
I'm worried that multiple tests may be using this in parallel, let's move it into each test (even inside each t.Run()).